modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js

```diff
@@ -41,6 +42,7 @@ export default function DropdownMenu({
 					'dropdown-action': actionsDropdown,
 				})}
 				items={normalizeDropdownItems(items) || []}
+				menuElementAttrs={menuElementAttrs}
```

Comment:
Well, even though Clay uses this name which is something I think we got a little wrong in the naming because we have other components following the prefix of `*Props` this has left some of our components inconsistent we should improve this over time I would say in this scenario I prefer to go with `menuProps` to follow the same nomenclature as `additionalProps`. And looking at it now, it looks like we didn't use `additionalProps` 🤔

Reply:
> Well, even though Clay uses this name which is something I think we got a little wrong in the naming because we have other components following the prefix of *Props this has left some of our components inconsistent we should improve this over time I would say in this scenario I prefer to go with menuProps to follow the same nomenclature as additionalProps.

Makes sense, we could use menuProps here.

> And looking at it now, it looks like we didn't use additionalProps

I've checked and it is not being used anywhere in portal (only in the empty state taglib, that is adding all possible properties).

So I could:
- Remove additionalProps since it is not doing anything.
- Rename menuElementAttrs to menuProps.

Is that fine?

Reply:
**Update**: I've checked and additionalProps is there because is being [inherited from BaseContainerTag](https://github.com/liferay/liferay-portal/blob/cfb103e1fc9a570e943de4cfec73f2c41cc47a11/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/internal/servlet/taglib/BaseContainerTag.java#L447) so I guess we cannot remove it. I could reuse it instead of adding menuProps, but I think it might be a bit confusing because it is not clear if that attribute should be applied to the menu itself or the trigger element. What do you think?

Reply:
Oh I see, yeah I think we can just go with `menuProps` and leave the `additionalProps`.

Reply:
Updated! :partying_face: